core/__tests__/tasks/group/destroy.ts

```diff
@@ -45,7 +45,6 @@ describe("tasks/group:destroy", () => {
     });
 
```

Comment:
It might be helpful to add:
* The group will not be deleted if a destination is tracking this group
* The group will not be deleted if a destination is sending this group's memberships

Reply:
Added #1, but for #2 at least at the moment this isn't being checked and instead an `AfterDestroy` hook takes care of deleting those. Should this be changed?